addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -10677,12 +10675,12 @@ QUnit.module("Views", (hooks) => {
         });
         assert.containsOnce(
             target,
-            'img[data-src*="/web/image"][data-src$="&id=1&unique="]',
+            'img[data-src*="/web/image/partner/1/image"]',
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)